consensus/poet/simulator/simulator_rk_pub.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
If this needs to be checked in, can we provide some comments (ideally in the file, but also maybe next to it) about what this is and how it was generated?